cmd/kops/update_cluster.go

```diff
@@ -144,6 +144,22 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		}
 
 		glog.Infof("Using SSH public key: %v\n", c.SSHPublicKey)
+	} else {
+		items, err := keyStore.FindSSHPublicKeys(fi.SecretNameSSHPrimary)
+		if err != nil {
+			return fmt.Errorf("error finding SSH Key: %v", err)
+		}
+		if len(items) == 1 {
+			c.SSHPublicKey = string(items[0].Data)
```

Comment:
Wrote up some ideas here as part of issue review: https://github.com/kubernetes/kops/issues/1207#issuecomment-275897944

I punted to 1.5.1 because this is not trivial.  I'm leaning towards scanning the available public keys, but I just don't know if it is worth it, and thus it might be better to leave off `-i ~/.ssh/id_rsa` altogether.

The reason for the initial inclusion was that kube-up creates a new key, and this let people discover the fact that the key was their default key.